src/xrServerEntities/xrServer_Space.h

```diff
@@ -30,7 +30,7 @@ enum EPOType
 using DWORD_VECTOR = xr_vector<u32>;
 using BOOL_VECTOR = xr_vector<bool>;
 using FLOAT_VECTOR = xr_vector<float>;
-using LPSTR_VECTOR = xr_vector<LPSTR>;
+using LPSTR_VECTOR = xr_vector<pstr>;
```

Comment:
Let's leave it as is for now.
```suggestion
using LPSTR_VECTOR = xr_vector<LPSTR>;
```